lib/matplotlib/tests/test_pickle.py

```diff
@@ -150,7 +150,15 @@ def test_pickle_load_from_subprocess(fig_test, fig_ref, tmp_path):
     proc = subprocess_run_helper(
         _pickle_load_subprocess,
         timeout=60,
-        extra_env={'PICKLE_FILE_PATH': str(fp), 'MPLBACKEND': 'Agg'}
+        extra_env={
+            "PICKLE_FILE_PATH": str(fp),
+            "MPLBACKEND": "Agg",
+            # subprocess_run_helper will set SOURCE_DATE_EPOCH=0, so for a dirty tree,
+            # the version will have the date 19700101. As we aren't trying to test the
+            # version compatiblity warning, force setuptools-scm to use the same
```

Comment:
```suggestion
            # version compatibility warning, force setuptools-scm to use the same
```
should fix pre-commit

Reply:
Weird, I thought I usually get an email about that... Maybe on a different repo?